rpms/base.pan

```diff
@@ -28,8 +25,8 @@ variable OS_CORE_RDMA_ENABLED ?= false;
     pkg_repl('mcelog');
     pkg_repl('microcode_ctl');
     pkg_repl('NetworkManager');
-    # network-scripts is required by ncm-network 21.4.0
-    pkg_repl('network-scripts');
+    # ncm-network relies on ifcfg scripts
+    pkg_repl('NetworkManager-initscripts-updown');
```

Comment:
While this is true, we're not supporting the `initscripts` back-end on EL9, so this isn't the correct justification.